package.json

```diff
@@ -71,11 +71,12 @@
     "node": ">= 8.9.0"
   },
   "dependencies": {
-    "apollo-cache-inmemory": "^1.3.10",
-    "apollo-client": "^2.4.6",
-    "apollo-link": "^1.2.3",
-    "apollo-link-http": "^1.5.5",
-    "apollo-link-schema": "^1.1.1",
+    "apollo-cache-inmemory": "^1.3.11",
+    "apollo-client": "^2.4.7",
+    "apollo-link": "^1.2.4",
+    "apollo-link-http": "^1.5.7",
+    "apollo-link-schema": "^1.1.2",
+    "graphql": "git://github.com/graphql/graphql-js.git#npm",
```

Comment:
Do we want to pin this to a version range?

Reply:
It has to be specifically the github version, or flow errors out